src/sections/attachments.css

```diff
@@ -13,6 +13,17 @@
     fill: #008898;
 }
 
+.attach-documents-table-icon-remove {
```

Comment:
This is a duplicate selector, it has the same CSS rules as the selector **.attach-documents-table-icon**
Please merge them into one selector.